sycl/test-e2e/Graph/Explicit/event_status_querying.cpp

```diff
@@ -0,0 +1,146 @@
+// REQUIRES: level_zero, gpu
+// RUN: %{build} -o %t.out
+// RUN: %{run} %t.out 2>&1 | FileCheck %s
+//
+// CHECK-DAG: complete
```

Comment:
Think this can be `CHECK` rather than `CHECK-DAG`, since there's no other checks to worry about the ordering of. Same comment for RecordReplay test.